airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -29,51 +29,39 @@ class Config:
 
     name: str = Field(
         title="Name",
-        description="The name value of insight",
     )
 
     fields: Optional[List[ValidFields]] = Field(
         title="Fields",
-        description="A list of chosen fields for fields parameter",
         default=[],
     )
 
     breakdowns: Optional[List[ValidBreakdowns]] = Field(
         title="Breakdowns",
-        description="A list of chosen breakdowns for breakdowns",
         default=[],
     )
 
     action_breakdowns: Optional[List[ValidActionBreakdowns]] = Field(
         title="Action Breakdowns",
-        description="A list of chosen action_breakdowns for action_breakdowns",
         default=[],
     )
 
     time_increment: Optional[PositiveInt] = Field(
         title="Time Increment",
-        description=(
-            "Time window in days by which to aggregate statistics. The sync will be chunked into N day intervals, where N is the number of days you specified. "
-            "For example, if you set this value to 7, then all statistics will be reported as 7-day aggregates by starting from the start_date. If the start and end dates are October 1st and October 30th, then the connector will output 5 records: 01 - 06, 07 - 13, 14 - 20, 21 - 27, and 28 - 30 (3 days only)."
-        ),
+        description="The field is a time interval in days to calucate statistics. We will split the sync into N-days time frames, where N is the number of defined in this field.",
         exclusiveMaximum=90,
         default=1,
     )
 
     start_date: Optional[datetime] = Field(
         title="Start Date",
-        description="The date from which you'd like to replicate data for this stream, in the format YYYY-MM-DDT00:00:00Z.",
         pattern=DATE_TIME_PATTERN,
         examples=["2017-01-25T00:00:00Z"],
     )
 
     end_date: Optional[datetime] = Field(
         title="End Date",
-        description=(
-            "The date until which you'd like to replicate data for this stream, in the format YYYY-MM-DDT00:00:00Z. "
-            "All data generated between the start date and this date will be replicated. "
-            "Not setting this option will result in always syncing the latest data."
-        ),
+        description="The field is the date until which to pull data. If not specified,  we will use the latest data available.",
```

Comment:
@am6010 there are multiple spaces here
`If not specified,  we will use the latest data available.`